.github/workflows/feg-workflow.yml

```diff
@@ -0,0 +1,148 @@
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action feg-precommit update failed"
+          SLACK_USERNAME: "Feg workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+  feg-build:
+    needs: feg-precommit
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      DOCKER_REGISTRY: "${{ secrets.DOCKER_REGISTRY }}"
+      DOCKER_USERNAME: "${{ secrets.DOCKER_USERNAME }}"
+      DOCKER_PASSWORD: "${{ secrets.DOCKER_PASSWORD }}"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+      - name: generate test certs and snowflake
+        run: |
+            # TODO add rootCA.pem and snowflake files in the ubuntu-1604:201903-01 image
+            # create directories
+            cd ${MAGMA_ROOT} && mkdir -p .cache/test_certs/ && mkdir -p .cache/feg/
+            # create test certs
+            cd ${MAGMA_ROOT}/.cache/test_certs/
+            openssl genrsa -out rootCA.key 2048
+            openssl req -x509 -new -nodes -key rootCA.key -sha256 -days 365000 -out rootCA.pem -subj "/C=US/CN=rootca.magma.test"
+            # create snowflake
+            cd ${MAGMA_ROOT}/.cache/feg/ && touch snowflake
+      - name: Export Docker registry
+        run: |
+            cd ${MAGMA_ROOT}/feg/gateway/docker
+            echo "DOCKER_REGISTRY=feg_ python3 build.py" >> $GITHUB_ENV
+      - name: run docker containers and check health
+        run: |
+            cd ${MAGMA_ROOT}/feg/gateway/docker
+            echo "DOCKER_REGISTRY=feg_ python3 build.py -e" >> $GITHUB_ENV
+      - name: Tag and push to Docker Registry
+        run: |
+            ./ci-scripts/tag-push-docker.sh -i 'gateway_go|gateway_python' -t '${GITHUB_SHA:0:8}' -tl false -p feg
+      - name: persist-githash-version - Create version file
+        run: |
+            cd ${MAGMA_ROOT}/circleci
+            mkdir -p versions
+            echo "${GITHUB_SHA:0:8}" > versions/feg_version
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "Github action feg-precommit update failed"
+          SLACK_USERNAME: "Feg workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+      # Notify ci channel when push succeeds
+      - name: Notify success to slack
+        if: success() && github.ref == 'refs/heads/master'
+        uses: rtCamp/action-slack-notify@v2.0.2
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
```

Comment:
You must specify this WEBHOOK secret: SLACK_WEBHOOK_OSS
We are publishing successful artifact upload to ci-artifact channel

Reply:
Ah!.. thank you! let me fix that.